shogun-lib/src/main/java/de/terrestris/shogun/lib/security/SecurityContextUtil.java

```diff
@@ -80,19 +79,15 @@
      * @return
      */
     public Optional<User> getUserFromAuthentication(Authentication authentication) {
-        final Object principal = authentication.getPrincipal();
-        if (!(principal instanceof KeycloakPrincipal)) {
-            return Optional.empty();
-        }
-        // get user info from authentication object
-        String keycloakUserId = getKeycloakUserIdFromAuthentication(authentication);
+        String keycloakUserId = this.getKeycloakUserUuid(authentication);
```

Comment:
do we still need `getKeycloakUserIdFromAuthentication` ?

maybe we can mark it deprecated, or just replace the implementation and keep the name if it does not have any benefit over `getKeycloakUserUuid`

Reply:
Yes it's still used in `getUserBySession`